CONTRIBUTORS.md

```diff
@@ -0,0 +1,6 @@
+### **Contributors:**
```

Comment:
Aren't we the Development Lead?
and there is no any contributor yet?

As the example shown in 
https://github.com/ttimbers/convertempPy/blob/master/CONTRIBUTORS.md